### PR TITLE
Initial tested version of base historian for VOLTTRON modular code base

### PR DESCRIPTION
**Pull request by schandrika**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: ci
+
+on:
+  push:
+    branches:
```
Comment (craigpnnl) [inline .github/workflows/ci.yml]: remove this file

**Review by craigpnnl — CHANGES_REQUESTED** [review]
Add run-tests.yml file from https://github.com/eclipse-volttron/github-tooling/tree/main/actions-all-repositories

Add auto-assign as well

**Comment by craigpnnl** [comment]
Don't forget to resolve the conflicts please

**Comment by schandrika** [comment]
@craig8 I resolved conflicts and removed ci.yml

**Comment by craigpnnl** [comment]
remove ci.yml and add the other yml files for workflows please.

**Pull request merged**